docs/gaia/gaia.rst

```diff
@@ -320,49 +359,48 @@ Query saving results in a file (you may use 'output_format' to specified the res
 available formats are: 'votable', 'votable_plain', 'fits', 'csv' and 'json', default is 'votable'):
 
 .. code-block:: python
+.. doctest-remote-data::
 
   >>> from astroquery.gaia import Gaia
   >>> job = Gaia.launch_job("select top 100 "
   ...                       "solution_id,ref_epoch,ra_dec_corr,astrometric_n_obs_al, "
   ...                       "matched_observations,duplicated_source,phot_variable_flag "
   ...                       "from gaiadr2.gaia_source order by source_id",
   ...                       dump_to_file=True, output_format='votable')
-  >>> print(job.outputFile)
-
+  >>> print(job.outputFile)     # doctest: +IGNORE_OUTPUT
   1592474300458O-result.vot.gz
-
   >>> r = job.get_results()
-  >>> print(r['solution_id'])
-
+  >>> print(r['solution_id'])  # doctest: +IGNORE_OUTPUT
     solution_id
   -------------------
-  1635378410781933568
-  1635378410781933568
-  1635378410781933568
-  1635378410781933568
+  1635721458409799680
+  1635721458409799680
+  1635721458409799680
+  1635721458409799680
+  1635721458409799680
                 ...
   Length = 100 rows
 
 Note: you can inspect the status of the job by typing:
 
 .. code-block:: python
+.. doctest-remote-data::
 
-  >>> print(job)
-
+  >>> print(job)     # doctest: +IGNORE_OUTPUT
```

Comment:
It looks like this is being ignored because the automatically generated output file name changes from run to run, but wouldn't it be better to provide a filename to the `launch_job` query above?